modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+The current release adds the following improvement:
+
+* With this update, you can selectively disable the `redis` and `application-controller` components for an Argo CD instance in a specified namespace. These components are enabled by default.
+To disable a component, set the `enabled` flag to `false` in the `.spec.<component>.enabled` field of the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-3723[GITOPS-3723]
++
+.Example: Disabling the `redis` and `application-controller` components
+[source, yaml]
+----
+spec:
+  controller:
+    enabled: false
+  redis:
+    enabled: false
+----
++
+[NOTE]
+====
+This feature is currently limited to the `redis` and `application-controller` components. Support for other components is expected to be included in the forthcoming {gitops-title} v1.11.1 release.
+====
+
+[id="fixed-issues-1-11-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, the Argo CD Notifications Controller did not support custom certificates added to `argocd-tls-certs-cm` config map. As a result, notification services with custom certificates did not receive notifications due to the "x509: certificate signed by unknown authority" error. This update fixes the issue by correctly initializing the cert resolver function in the Argo CD Notifications Controller to load all certificates stored in `argocd-tls-certs-cm` config map. Now, notification services with custom certificates can successfully receive notifications. link:https://issues.redhat.com/browse/GITOPS-2809[GITOPS-2809]
+
+* Before this update, users would face `PrometheusOperatorRejectedResources` alerts when the {gitops-title} Operator was not installed in the `openshift-gitops-operator` namespace. The problem affected users who upgraded from earlier versions of the {gitops-title} Operator to v1.10. This update fixes the issue by updating the Operator's `serverName` metrics service to reflect the correct installation namespace. Now, users who upgrade or install the {gitops-title} Operator in namespaces other than `openshift-gitops-operator` should not see these alerts. link:https://issues.redhat.com/browse/GITOPS-3424[GITOPS-3424]
```

Comment:
```suggestion
* Before this update, users would face `PrometheusOperatorRejectedResources` alerts when the {gitops-title} Operator was not installed in the `openshift-gitops-operator` namespace. The problem affected users who upgraded from earlier versions of the {gitops-title} Operator to v1.10. This update fixes the issue by updating the Operator's `serverName` metrics service to reflect the correct installation namespace. Now, users who upgrade or install the {gitops-title} Operator in namespaces other than `openshift-gitops-operator` cannot see these alerts. link:https://issues.redhat.com/browse/GITOPS-3424[GITOPS-3424]
```

Reply:
Can we use "cannot" instead of "should not"? I think "cannot" provides clarity

Reply:
IMO I feel "should not" sounds more natural and is more suitable for the described situation.